models/articles.models.js

```diff
@@ -11,3 +11,18 @@ exports.fetchArticleById = (article_id) => {
       }
     });
 };
+
+exports.changeArticleById = (article_id, inc_votes) => {
+  return db
+    .query(
+      "UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *",
+      [inc_votes, article_id]
+    )
+    .then((result) => {
+      if (result.rows.length > 0) {
```

Comment:
Another cool way to write something like this is `if(result.rows.length)`, as 0 is falsy.